easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_libpath.patch

```diff
@@ -0,0 +1,59 @@
+Fix include path for jbig libraries
+Use CC, CFLAGS from EB.
+Drop bad make depend lines.
+
+John Dey, 2019-05-29  jfdey@fredhutch.org  fizwit@github.com
+diff -ruN jbigkit-2.1.orig/libjbig/Makefile jbigkit-2.1/libjbig/Makefile
+--- jbigkit-2.1.orig/libjbig/Makefile	2014-03-27 11:47:15.000000000 -0700
++++ jbigkit-2.1/libjbig/Makefile	2019-06-14 14:19:21.901910000 -0700
+@@ -1,10 +1,7 @@
+ # Unix makefile for the JBIG-KIT library
+ 
+-# Select an ANSI/ISO C compiler here, GNU gcc is recommended
+-CC = gcc
+-
+ # Options for the compiler: A high optimization level is suggested
+-CFLAGS = -g -O -W -Wall -ansi -pedantic # --coverage
++CFLAGS += -g -O -W -Wall -ansi -pedantic # --coverage
```

Comment:
I am a little confused here. The comment says that a high optimization level is suggested, but it uses only `-O` which is not such a high optimization IMHO. As a matter of fact in `jbigkit-2.1/Makefile` they set it as `-O2`.
What do you think about getting rid of the `-O` and let it use the `-O2` that EasyBuild sets?